pkg/views/provider/select.go

```diff
@@ -4,16 +4,26 @@
 package provider
 
 import (
+	"sort"
+
 	"github.com/charmbracelet/bubbles/list"
 	tea "github.com/charmbracelet/bubbletea"
 	"github.com/daytonaio/daytona/pkg/apiclient"
 	"github.com/daytonaio/daytona/pkg/common"
 	"github.com/daytonaio/daytona/pkg/views"
 )
 
+type ProviderView struct {
+	Name      string
+	Version   string
+	Installed *bool
```

Comment:
Why is this a pointer?

Reply:
because most GetProviderFromPrompt calls have/need no information about whether the provider is installed

it is only used during daytona target set, otherwise nil